core/baseline-core-v1.0-psd01.md

```diff
@@ -637,36 +637,418 @@ Describes the mechanisms ensuring that other parties (i.e parties outside of tra
+```
+
+See sections Xx and YY below on BPI accounts associated with state objects and BPI transactions for details.
+
+**[RXX]** A workstep instance MUST be associated with only one BPI workgroup.
+
+**[RXX]** A workstep instance MUST inherit the security and privacy policies of its associated workgroup.
+
+See the details of workgroups and their security and privacy policies below.
+
+**[RXX]** A workstep MUST have a unique identifier within a BPI.
+
+**[RXX]** A workstep MUST be updatable.
+
+**[RXX]** A workstep instance MUST NOT be updated while the workstep is being executed by the BPI.
+
+This ensures that no breaking changes with potentially significant negative business impact are introduced while a workstep instance is being executed.
+
+**[RXX]** A workstep MUST be versioned within a BPI.
+
+Note that versions of the same workstep do not have to be compatible with one another. 
+
+**[RXX]** A workstep MUST be executed by a BPI.
+
+**[RXX]** The input of a workstep MUST be submitted by an authorized member of the workgroup attached to that workstep.
+
+Note, that this allows for delegation of authorization from the authorization bearing Entity A to Entity B, akin to a power-of-attorney. This concept is also known as attenuated authorization.
+
+**[RXX]** A workstep MUST be deterministic.
+
+This means that for a given input, there can be only one valid output from the workstep generated by the BPI.
+
+**[RXX]** The output from a workstep execution MUST be finalized through an agreed upon quorum of cryptographic signatures of the workgroup participants associated with the workstep.
+
+This means that the output of a workstep execution must be verified and agreed upon by a previously defined number of the workgroup participants. This naturally extends to the input as well. 
+
+**[RXX]** The output from a workstep execution MUST be a valid zero-knowledge proof of correctness of the input generated by the BPI executing the workstep (privacy preservation).
+
+A Zero-Knowledge Proof is defined as having to satisfy the following three properties:
+
+* **Completeness:** if the statement is true,  an honest verifier, i.e., an entity following the protocol properly, will be convinced of this fact by an honest prover.
+* **Soundness:** if the statement is false, no cheating prover can convince an honest verifier that it is true, except with some small probability.
+* **Zero-Knowledge:** if the statement is true, no verifier learns anything other than the fact that the statement is true. In other words, just knowing the statement (not the secret) is sufficient t construct a scenario that shows that the prover knows the secret. This is formalized by showing that every verifier has some simulator that, given only the statement to be proved (and no access to the prover), can produce a transcript that "looks like" an interaction between the honest prover and the verifier.
```

Comment:
Is this a typo? `sufficient to construct` instead of `sufficient t construct`

Reply:
yes. thank you. corrected.